droopler.install

```diff
@@ -1455,3 +1455,16 @@ function droopler_update_8117() {
   $updater->checklist()
     ->saveProgress(['v2.2' => $values]);
 }
+
+/**
+ * Droopler 2.2 update configs for modules.
```

Comment:
to już trzeci z kolei `Droopler 2.2 update configs for modules.` Może postarajmy się nadać im jakieś bardziej konkretne nazwy?